dotnetv3/AutoScaling/Scenarios/AutoScalingBasics/AutoScalingBasics.cs

```diff
@@ -0,0 +1,224 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[AutoScaling.dotnetv3.AutoScalingBasics]
+
+using Amazon.EC2;
+using Microsoft.Extensions.Configuration;
+
+namespace AutoScalingBasics;
+
+public class AutoScalingBasics
+{
+    private static ILogger logger = null!;
+
+    static async Task Main(string[] args)
+    {
+        // Set up dependency injection for the Amazon service.
+        using var host = Host.CreateDefaultBuilder(args)
+            .ConfigureLogging(logging =>
+                logging.AddFilter("System", LogLevel.Debug)
+                    .AddFilter<DebugLoggerProvider>("Microsoft", LogLevel.Information)
+                    .AddFilter<ConsoleLoggerProvider>("Microsoft", LogLevel.Trace))
+            .ConfigureServices((_, services) =>
+            services.AddAWSService<IAmazonAutoScaling>()
+                .AddAWSService<IAmazonCloudWatch>()
+                .AddAWSService<IAmazonEC2>()
+                .AddTransient<AutoScalingWrapper>()
+                .AddTransient<CloudWatchWrapper>()
+                .AddTransient<EC2Wrapper>()
+                .AddTransient<UIWrapper>()
+            )
+            .Build();
+
+        logger = LoggerFactory.Create(builder => { builder.AddConsole(); })
+            .CreateLogger<AutoScalingBasics>();
+
+        var autoScalingWrapper = host.Services.GetRequiredService<AutoScalingWrapper>();
+        var cloudWatchWrapper = host.Services.GetRequiredService<CloudWatchWrapper>();
+        var ec2Wrapper = host.Services.GetRequiredService<EC2Wrapper>();
+        var uiWrapper = host.Services.GetRequiredService<UIWrapper>();
+
+        var configuration = new ConfigurationBuilder()
+            .SetBasePath(Directory.GetCurrentDirectory())
+            .AddJsonFile("settings.json") // Load test settings from .json file.
+            .AddJsonFile("settings.local.json",
+                true) // Optionally load local settings.
+            .Build();
+
+        var imageId = configuration["ImageId"];
+        var instanceType = configuration["InstanceType"];
+        var launchTemplateName = configuration["LaunchTemplateName"];
+
+        // The name of the Auto Scaling group.
+        var groupName = configuration["GroupName"];
+
+        // The Amazon Resource Name (ARN) of the AWS Identity and Access Management (IAM) service-linked role.
+        var serviceLinkedRoleARN = configuration["ServiceLinkedRoleArn"];
+
+        uiWrapper.DisplayTitle("Auto Scaling Basics");
+        uiWrapper.DisplayAutoScallingBasicsDescription();
```

Comment:
Should this be DisplayAutoScalingBasicsDescription? (one L )

Reply:
Name was corrected. Thanks for noticing.